upcgen.sh

```diff
@@ -0,0 +1,40 @@
+package: Upcgen
+version: "%(tag_basename)s"
+tag: tag-15-05-22-3
+source: https://github.com/nburmaso/upcgen
```

Comment:
is this your private fork of Upcgen ?

I believe that if a fork is necessary, we should have an official ALICE version under alisw. But @ktf @TimoWilken may further clarify the rules.

Reply:
Hi Sandro. This is the main repository of Upcgen, there is no official ALICE version of it. Should I prepare one?

Reply:
Hi @nburmaso. Are you OK with transferring the nburmaso/upcgen repo to the alisw organisation? This will update its name to alisw/upcgen. I will give you full admin access to the alisw/upcgen repo in that case.

I've sent you an invitation to the alisw GitHub organisation -- please check your emails for the link to accept the invitation or go to <https://github.com/alisw>. When you've accepted it, you should have the rights to transfer the repo over -- go to your repo's settings and at the bottom, click the "transfer" button.

If transferring is not an option, please let me know and we can work out an alternative solution.

Reply:
Hi @TimoWilken. Thank you for the suggestion, but I would like to keep the main repository as a personal one. Can I make a mirror of it inside `alisw` for the recipe?

Reply:
OK, you should be able to fork it yourself. If this case, though, any tags that you refer to in the alidist recipe must exist in the alisw/upcgen repo (whether they exist in your personal repo or not).

Reply:
Ok, sounds good to me. I have created a fork and switched to it in the recipe.